src/assets/styles/main.scss

```diff
@@ -257,18 +228,36 @@ $icon-font-path: '../icon-font' !default;
     }
   }
 
+  .vjs-control {
+    width: 2.5em;
+  }
+
+  .vjs-control::before,
+  .vjs-icon-placeholder:before {
+    font-size: 1.8em;
+    line-height: 1.7;
+    -moz-osx-font-smoothing: grayscale;
+    -webkit-font-smoothing: antialiased;
+    text-rendering: optimizeLegibility;
```

Comment:
why would you need these 3? or they were already here and this just moved from a different place?

Reply:
this fix the icon-font smoothness/sharpness, it is not standard but works.